### PR TITLE
update indentation in bcc/tarjan

### PR DESCRIPTION
**Pull request by neatudarius**
update indentation in bcc/tarjan

### CODE REVIEW

File: lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp [hunk 1]
```diff
@@ -15,253 +15,260 @@ using namespace std;
...
 
-  // adj[node] lista de adiacenta a nodului node pentru graful initial
-  vector<int> adj[NMAX];
+    // adj[node] lista de adiacenta a nodului node pentru graful initial
+    vector<int> adj[NMAX];
 
-  // bcc[i] = componenta biconexa cu indicele i
-  vector<vector<int>> bcc;
+    // bcc[i] = componenta biconexa cu indicele i
+        vector<vector<int>> bcc;
```
Comment (mirub) [inline lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp:33]:
```suggestion
    vector<vector<int>> bcc;
```

File: lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp [hunk 2]
```diff
@@ -15,253 +15,260 @@ using namespace std;
...
-      adj[y].push_back(x);
+            adj[x].push_back(y);
+            adj[y].push_back(x);
+        }
     }
-  }
 
-  void get_result() { tarjan(); }
+    void get_result() { 
+        tarjan(); }
```
Comment (mirub) [inline lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp:79]: Please fix the indentation.

File: lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp (hunk 1 above)
Comment (neatudarius) [reply lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp:33]: Ok

File: lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp [hunk 3]
```diff
@@ -15,253 +15,259 @@ using namespace std;
...
+    // n = numar de noduri, m = numar de muchii
+    int n, m;
 
-  // adj[node] lista de adiacenta a nodului node pentru graful initial
-  vector<int> adj[NMAX];
+    // adj[node] lista de adiacenta a nodului node pentru graful initial
+    vector<int> adj[NMAX];
 
-  // bcc[i] = componenta biconexa cu indicele i
-  vector<vector<int>> bcc;
```
Comment (RaduNichita) [inline lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp:32]: Aici ar trebui sa fie un std in fata la vector

File: lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp [hunk 4]
```diff
@@ -15,253 +15,259 @@ using namespace std;
...
 private:
-  // n = numar de noduri, m = numar de muchii
-  int n, m;
+    // n = numar de noduri, m = numar de muchii
+    int n, m;
 
-  // adj[node] lista de adiacenta a nodului node pentru graful initial
-  vector<int> adj[NMAX];
+    // adj[node] lista de adiacenta a nodului node pentru graful initial
+    vector<int> adj[NMAX];
```
Comment (CristianOlaru16) [inline lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp:30]: please use std vector

File: lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp [hunk 5]
```diff
@@ -15,253 +15,260 @@ using namespace std;
...
```
Comment (mzzmara) [inline lab/skel_graph/traversal/_examples/bcc/tarjan/main.cpp:30]: Please use "std" in front of every STL structure.

**Pull request merged**